functions.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * connects to the database and extracts all items required and returns in an associative array
+ * @param $dbName, the name of the db to connect to
+ *
+ * @return array, the array of items in the collection
+ */
+function connect_to_db_and_extract_items($dbName)
```

Comment:
a function should only do one thing. if you need an "and" in the function name, its a good indicator that it should be two functions